specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -79,6 +79,55 @@
         }
       }
     },
+    "/keys/{key-name}/roll": {
+      "put": {
+        "tags": [
+          "Keys"
+        ],
+        "operationId": "RotateKey",
+        "summary": "Creates a new key version, stores it, then returns key parameters, attributes and policy to the client.",
+        "description": "The operation will rotate the key based on the key policy. It requires the keys/rotate permission.",
+        "parameters": [
+          {
+            "name": "key-name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[0-9a-zA-Z-]+$",
+            "description": "The name for the new key. The system will generate the version name for the new key."
+          },
+          {
+            "name": "force",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "description": "Key rotation will happen immediately when it is true. Otherwise, the api will compute current key policy to make key rotation decision. The value will be true if not specified."
```

Comment:
Could you elaborate on this? It is based on, say, the 90'th day if 90D is the lifetime action? Maybe if not here, at least be sure to file an issue to improve docs. I can imagine there being some questions on this.

/cc @jlichwa